internal/k8s/ako_init.go

```diff
@@ -162,7 +162,7 @@ func delConfigFromData(data map[string]string) bool {
 	return delConf
 }
 
-func deleteConfigFromConfigmap(cs kubernetes.Interface) bool {
+func DeleteConfigFromConfigmap(cs kubernetes.Interface) bool {
```

Comment:
revert change, not used

Reply:
we are using this in our clean stale vs code.